ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -33,8 +35,46 @@
     }),
 
     computed: {
+      ...mapGetters('interactives', [
+        'currentInteractiveDataLoading',
+        'currentInteractive',
+        'currentInteractiveSession'
+      ]),
+
+      interactiveConfig () {
+        switch (this.currentInteractive.interactiveType) {
+        case 'draggable-statement-completion':
+          return this.currentInteractive.draggableStatementCompletionData
+        case 'insert-code':
+          return this.currentInteractive.insertCodeData
+        case 'draggable-ordering':
+          return this.currentInteractive.draggableOrderingData
+        default:
+          // TODO handle_error_ozaria
+          noty({ text: 'Interactive type is not set for the interactive', type: 'error', timeout: '2000' })
+          throw new Error('Invalid interactive type provided for interactive')
+        }
+      },
+
+      localizedInteractiveConfig () {
+        const interactiveConfigI18n = this.interactiveConfig.i18n || {}
```

Comment:
@shubhi1092 picking this conversation up https://github.com/codecombat/codecombat/pull/5384#discussion_r290069254

I'll add this to ozariaUtils once you give this code the thumbs up

Reply:
Sounds good, I think I only have the same question as before - i.e. Will this also handle the i18n for interactive root properties, i.e. for promptText (https://github.com/codecombat/codecombat/blob/master/app/schemas/models/interactives/interactive.schema.js#L23), or just the interactive data?

Reply:
^ Same question, + can we move this to ozariaUtils

Reply:
Moved to ozariaUtils and integrated with interactives.

To answer your question: the util method is a general implementation of our i18n logic and it is applied to the top level config and the nested config as necessary.